datasets/standvastgoed/dataset.json

```diff
@@ -215,61 +221,50 @@
             "type": "string",
             "provenance": "$.status.omschrijving",
             "title": "nummeraanduidingStatusOmschrijving",
-            "shortname": "nagStatusOms",
+            "shortname": "nagStatusOmschrijving",
             "description": "Omschrijving voor de fase van de levenscyclus van een adresseerbaar object verblijfsobject, waarin het zich bevindt"
           },
           "adres27": {
             "type": "string",
             "title": "adres27",
-            "description": "Combinatie van adres kenmerken (27 karakters lang): openbareRuimteIdentificatie, huisnummer, huisletter en huisnummertoevoeging"
+            "description": "Combinatie van adres kenmerken (27 karakters lang): openbareruimteIdentificatie, huisnummer, huisletter en huisnummertoevoeging"
           },
           "adres17": {
             "type": "string",
             "title": "adres17",
             "description": "Combinatie van adres kenmerken (17 karakters lang): postcode, huisnummer, huisletter en huisnummertoevoeging"
           },
           "gebruiksdoel": {
-            "type": "array",
-            "items": {
-              "type": "object",
-              "properties": {
-                "code": {
-                  "type": "string"
-                },
-                "omschrijving": {
-                  "type": "string"
-                }
-              }
-            },
+            "type": "string",
             "title": "gebruiksdoel",
             "description": "Een categorisering van de gebruiksdoelen van het betreffende verblijfsobject, zoals dit door de overheid als zodanig is toegestaan."
           },
           "gebruiksdoelWoonfunctieCode": {
             "type": "integer",
             "provenance": "$.gebruiksdoelWoonfunctie.code",
             "title": "gebruiksdoelWoonfunctieCode",
-            "shortname": "gebruiksdoelWoonCode",
+            "shortname": "gebruiksdoelWoonfunctieCode",
```

Comment:
Als de shortname gelijk is aan de naam in de dataset table, kan deze beter weggelaten worden.